### PR TITLE
added changes by Yunus

### PR DESCRIPTION
**Pull request by Yunuscanko**

### CODE REVIEW

File: spect_jitter.cc [hunk 1]
```diff
@@ -18,7 +19,11 @@ int main (){
   csv_reader csv;
   /* jitter info container */
   source_jitter jitter;
+  try{
   csv.read(TString(PATH+fileName).Data(), jitter, TString("Single"));
+  }catch (int a){
```
Comment (osahin) [inline spect_jitter.cc:24]: Please be explicit in the naming. What is 'a'?

File: src/csv_reader.cc [hunk 2]
```diff
@@ -10,8 +10,14 @@
 bool csv_reader::read( const std::string & fileName, source_jitter & sj, TString sampleTitle){
   std::cout << fileName << std::endl;
   std::fstream csv_file(fileName);
-    if(!csv_file) return false;
-    std::vector<double> jitterValues(2,0);
+  //error mesage in case of mis-named file
+  if(!csv_file){
+    std::cout <<  "Error. No such file exists."<< std::endl;
+    throw 5;
```
Comment (osahin) [inline src/csv_reader.cc:16]: Why 5, could you  define a constant with a more reasonable naming?

File: src/csv_reader.cc [hunk 3]
```diff
@@ -10,8 +10,14 @@
...
+
+ return false;
```
Comment (osahin) [inline src/csv_reader.cc:18]: Indentation is wrong

**Comment by Yunuscanko** [comment]
closing for retry

**Pull request closed without merging**